ci/prow/main.sh

```diff
@@ -1,4 +1,71 @@
 #!/bin/bash
-echo "Prow script initiated."
-oc whoami
-echo "Prow complete."
+read -r GITHUB_TOKEN </var/run/cred/operator_bundle_bot_github_token
+export GITHUB_TOKEN
+start_time="$(date +%s)"
+deadline="$((start_time+1800))"
+
+fail() {
+    echo "$@"
+    gh pr edit "${pr_url}" \
+        --remove-label "ocp/${OCP_CLUSTER_VERSION}/start" \
```

Comment:
I would also include skip label removal in case of tekton failure
```suggestion
        --remove-label "ocp/${OCP_CLUSTER_VERSION}/start" \
        --remove-label "ocp/${OCP_CLUSTER_VERSION}/skip" \
```

Reply:
If we get a /skip label, this script is pretty much a no-op, so this function is never going to be called.
The idea is that the trigger-prow-tests task clears all pre-existing ocp/* labels before adding the appropriate labels to trigger prow.